bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/views/OrderManagementView.java

```diff
@@ -0,0 +1,939 @@
+	private Composite leftComposite;
+	private Label cartIcon;
+	private Label titleLabel;
+	private Label createdLabel;
+	private TableViewerColumn[] mainColumns;
+	private Label statusLabel;
+	private Label statusValue;
+
+	public TableViewer tableViewer;
+	private TableViewer orderTable;
+	private TableViewer completedTable;
+
+	private Action checkInAction;
+	private IOrder actOrder;
+	private OrderManagementActionFactory actionFactory;
+	private Button orderButton;
+	private boolean showDeliveredColumn = false;
+
+	private Composite dispatchedComposite;
+	private Label dispatchedLabelTitle;
+	private Label dispatchedLabelIcon;
+	private Label dispatchedLabelState;
+
+	private Composite bookedComposite;
+	private Label bookedLabelTitle;
+	private Label bookedLabelIcon;
+	private Label bookedLabelState;
+	private Composite completedContainer;
+	private boolean isUIUpdating = false;
+
+	@Inject
+	private IOrderService orderService;
+
+	public Composite getCompletedContainer() {
+		return completedContainer;
+	}
+
+	public void setCompletedContainer(Composite completedContainer) {
+		this.completedContainer = completedContainer;
+	}
+
+
+	private Composite rightListComposite;
+	private ScrolledComposite scrolledComposite;
+	private ScrolledComposite rightScrollComposite;
+
+	private boolean hasFocus = false;
+	private boolean showAllYears = false;
+
+	private final Image IMGCLEAR = Images.IMG_CLEAR.getImage();
+	private final Image TICKIMAGE = Images.IMG_TICK.getImage();
+	private final Image DELIVERY_TRUCK = Images.IMG_DELIVERY_TRUCK.getImage(ImageSize._75x66_TitleDialogIconSize);
+	private final Image EINBUCHEN = Images.IMG_IMPORT.getImage();
+	private final Image LIFERANT = Images.IMG_ACHTUNG.getImage();
+
+	private static final GridData FILL_HORIZONTAL = new GridData(SWT.FILL, SWT.CENTER, true, false);
+	private static final GridData FILL_BOTH = new GridData(SWT.FILL, SWT.FILL, true, true);
+	private static final GridData ALIGN_RIGHT = new GridData(SWT.RIGHT, SWT.CENTER, true, false);
+	private static final GridData FIXED_WIDTH_180 = new GridData(180, SWT.DEFAULT);
+	private static final GridData FIXED_WIDTH_120 = new GridData(120, 25);
+	private static final GridData FIXED_WIDTH_50 = new GridData(50, SWT.DEFAULT);
+	private Map<Integer, Boolean> expandedStates = new HashMap<>();
+
+
+	public GenericObjectDropTarget dropTarget;
+
+	@org.eclipse.e4.core.di.annotations.Optional
+	@Inject
+	public void reload(@UIEventTopic(ElexisEventTopics.EVENT_RELOAD) Class<?> clazz) {
+		if (IStock.class.equals(clazz)) {
+			if (actOrder != null) {
+				refresh();
+				loadOpenOrders();
+			}
+			if (orderTable != null) {
+				loadOpenOrders();
+			}
+
+		}
+	}
+	
+	private RefreshingPartListener focusListener = new RefreshingPartListener(this) {
+		@Override
+		public void partActivated(IWorkbenchPartReference partRef) {
+			if (isMatchingPart(partRef)) {
+				hasFocus = true;
+				String valueToSet = hasFocus ? OrderManagementView.class.getName() : null;
```

Comment:
Hier wird hasFocus gesetzt, dann kann die Prüfung ob es gesetzt ist in der nächsten Zeile weg gelassen werden.